src/components/ListItem.jsx

```diff
@@ -1,25 +1,68 @@
-import { updateItem } from '../api/firebase';
+import { useState } from 'react';
+import { updateItem, deleteItem } from '../api/firebase';
 import { isWithinLastDay } from '../utils/dates';
+import { ListItemDetails } from './ListItemDetails';
 import './ListItem.css';
 
-export function ListItem({ item, listId }) {
-	const { id, name } = item;
+export function ListItem({ item, listId, handleDeleteConfirmation }) {
+	const [showDetails, setShowDetails] = useState(false);
+	const { id, name, dateLastPurchased, dateNextPurchased, totalPurchases } =
+		item;
+
+	const lastPurchasedDate = dateLastPurchased?.toDate().toDateString();
+	const nextPurchasedDate = dateNextPurchased?.toDate().toDateString();
 
 	const handlePurchase = async (e) => {
 		if (e.target.checked) {
 			await updateItem(listId, item);
 		}
 	};
 
+	const handleShowDetails = () => {
+		setShowDetails(!showDetails);
+	};
+
+	const handleDeleteItem = async () => {
+		if (window.confirm(`Are you sure you want to delete ${name}?`)) {
+			const result = await deleteItem(listId, id);
+			result && handleDeleteConfirmation(name);
```

Comment:
If delete `does` fail the user won't see anything happen. Should we let them know if it does?

Reply:
I agree with Lyle on this.

Reply:
yes that's a good point.  when we were utilizing alert we had an error message in there but upon switching we forgot to handle it.